src/sidebar/store/modules/filters.js

```diff
@@ -46,7 +46,7 @@ import { createStoreModule } from '../create-store';
  */
 
 /**
- * @typedef {Record<FilterKey, FilterOption>|{}} Filters
+ * @typedef {Record<FilterKey, FilterOption>} Filters
```

Comment:
The `{}` here shouldn't have been needed. `{}` _is_ a valid empty "record", but in some cases you need to cast `{}` to the specific type of record, similar to how `[]` sometimes needs to be cast to a specific type of empty array.